src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -219,7 +222,10 @@ const useCloudEnvironmentPolling = googleProject => {
       const [newDisks, newRuntimes] = !!googleProject ? await Promise.all([
         Ajax(signal).Disks.list({ googleProject, creator: getUser().email, includeLabels: 'saturnApplication,saturnWorkspaceName' }),
         Ajax(signal).Runtimes.list({ googleProject, creator: getUser().email })
-      ]) : [[], []]
+      ]) : await Promise.all([
+        Promise.resolve([]),
+        workspaceId ? Ajax(signal).Runtimes.listV2AzureWithWorkspace(workspaceId, { creator: getUser().email }) : Promise.resolve([])
```

Comment:
> When is the terminal button supposed to be enabled? I'm finding that in my GCP workspace, it is enabled when Juptyer is still creating or paused. In that situation, it opens the create modal and it seems like I could create a new runtime. I don't know if this behavior is different from what is on dev, but I'm confused about how the terminal button is supposed to behave when Jupyter isn't actually in the running state.

Yeah, if you have a Jupyter environment and its not in the running state, it either
1) is enabled if the runtime will eventually be in an intractable state or (starting/creating)
2) if it can be started

There's also, confusingly, a known bug where the ComputeModal (for creating an env) pops up inappropriately on launch we are targetting for the next sprint https://broadworkbench.atlassian.net/browse/IA-2989